main.py

```diff
@@ -1,3 +1,4 @@
+import json
```

Comment:
General comments:
1. you should split the code to main, extract and convert-to-our json. notice that you use data files and create json intermediate files that belong to a specific format, they shouldn't be sitting in the main directory. instead create a folder for the extract code and its data.
2. document the code - so future Sigal won't be mad at you